app/src/main/res/values/strings.xml

```diff
@@ -181,6 +181,18 @@
     <string name="delete_empty_folders">Delete empty folders after deleting their content</string>
     <string name="allow_photo_gestures">Allow controlling photo brightness with vertical gestures</string>
     <string name="allow_video_gestures">Allow controlling video volume and brightness with vertical gestures</string>
+    <string name="change_default_playback_speed">Default video playback speed</string>
+    <string name="default_playback_speed_1">x0.25</string>
+    <string name="default_playback_speed_2">x0.5</string>
+    <string name="default_playback_speed_3">x0.75</string>
+    <string name="default_playback_speed_4">x1.0</string>
+    <string name="default_playback_speed_5">x1.25</string>
+    <string name="default_playback_speed_6">x1.5</string>
+    <string name="default_playback_speed_7">x1.75</string>
+    <string name="default_playback_speed_8">x2.0</string>
+    <string name="default_playback_speed_9">x3.0</string>
+    <string name="default_playback_speed_10">x4.0</string>
+    <string name="default_playback_speed_11">x5.0</string>
```

Comment:
1. It shouldn't be in strings. Numeric value should be provided and formatted by the code according to current user region, not hard-coded. Check how it's done in Fossify Music Player.
2. It should be `0.25x`, `0.5x` etc.